src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http3RequestStream.cs

```diff
@@ -88,7 +88,14 @@ public void Dispose()
             {
                 _disposed = true;
                 AbortStream();
-                _stream.Dispose();
+                if (_stream.WritesClosed.IsCompleted)
```

Comment:
Is `_stream.WritesClosed.IsCompleted` expected to be true most of the time for basic GET/POST requests (when we're not doing bidirectional streaming)?

Reply:
Yes.